app/views/sprint-6/book-and-manage/make-a-referral/find-an-intervention/find-multi-select.html

```diff
@@ -47,7 +47,7 @@ <h1 class="govuk-heading-xl">Find an intervention - Alex River</h1>
         <div class="govuk-form-group">
           <label for="needs" class="govuk-label">Criminogenic needs</label>
           <select id="needs" name="needs" class="govuk-select" multiple="multiple" data-auto-complete="true">
-            <option value="accommodation" selected>Accommodation</option>
+            <option value="accommodation" selected="selected">Accommodation</option>
```

Comment:
I'm pretty sure this is valid as it is - according to the spec `selected` is a boolean attribute, meaning its presence switches it on: https://html.spec.whatwg.org/multipage/form-elements.html#attr-option-selected

Reply:
You're right. I should have spent more time looking into it, it's weird the linter just does that... I'm gonna remove the commit

Reply:
What linter was it?

Reply:
Prettier (vs code add-on)